python/tests/timeSeries/readUnrTxyz2_test.py

```diff
@@ -1,7 +1,21 @@
 import sys
 sys.path.append("./src")
 
-import timeSeries as ts
+from timeSeries import timeSeries as ts
 
 def test_readUnrTxyz2():
-    assert True == True
+    series = ts.TimeSeries()
+    series.readUnrTxyz2("./tests/timeSeries/AREQ.IGS08.txyz2")
+
+    assert series.time.size == 7759
```

Comment:
pretty minor, but could the `7759` constant be pulled out into a variable for clarity on what it is?

Reply:
Yea prolly, I can make that change